ipa-core/src/query/runner/oprf_ipa.rs

```diff
@@ -147,13 +147,15 @@ where
         #[cfg(not(feature = "relaxed-dp"))]
         let padding_params = PaddingParameters::default();
         match config.per_user_credit_cap {
+            1 => oprf_ipa::<_, BA8, BA3, HV, BA20, 1, 256>(ctx, input, aws, dp_params, padding_params).await,
```

Comment:
Using BA3 for trigger values with cap 1 is gross, but the input is assumed to be fixed size, so it is a much bigger change to make if we want it